runtime/compiler/z/codegen/J9MemoryReference.cpp

```diff
@@ -393,8 +397,11 @@ J9::Z::MemoryReference::create(TR::CodeGenerator* cg, TR::Node* node)
 
       typedef J9::Z::UnresolvedDataReadOnlySnippet::CCUnresolvedData CCUnresolvedData;
 
-      CCUnresolvedData* ccUnresolvedDataAddress =
-         reinterpret_cast<J9::Z::UnresolvedDataReadOnlySnippet::CCUnresolvedData*>(cg->allocateCodeMemory(sizeof(CCUnresolvedData), false));
+      OMR::CCData *codeCacheData = cg->getCodeCache()->manager()->getCodeCacheData();
+      OMR::CCData::index_t index;
+      codeCacheData->put(NULL, sizeof(CCUnresolvedData), 8, NULL, index);
+      
+      CCUnresolvedData* ccUnresolvedDataAddress = codeCacheData->get<CCUnresolvedData>(index);
```

Comment:
You have to check the result of `put()` for failure, not `get()`.

Reply:
Fixed in https://github.com/eclipse/openj9/compare/e90116d7332681724cdd8aa492ff4522dadd0d6a..69c427858ff45f769e7f9831d4e4777159a0a758

Reply:
Checking result of `put` in https://github.com/eclipse/openj9/compare/69c427858ff45f769e7f9831d4e4777159a0a758..e6572d23ab0cf110a98cc239bea656e8aa8729ec